src/ontology/mondo-ingest.Makefile

```diff
@@ -28,6 +28,21 @@ dependencies:
 	$(MAKE) pip-sssom
 	$(MAKE) pip-semsql
 
+####################################
+### General ########################
+####################################
+%.db: %.owl
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+	@rm -f tmp/merged-relation-graph.tsv.gz
+	RUST_BACKTRACE=full semsql make tmp/$*.db -P config/prefixes.csv
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+	@rm -f tmp/merged-relation-graph.tsv.gz
+
+tmp/mondo-ingest.db: mondo-ingest.db
```

Comment:
## Add: `tmp/mondo-ingest.db`
This is needed by the synchronization pipeline, which hasn't been merged yet. But since we're sorting out other `.db` goal idiosyncrasies in this PR, I think it'd good to do here and I can rebase into my PR after.

The reason this one needs a specific goal and can't be handled by `%.db` is because the `%.db` goal (and SemanticSQL by design) expect that the directory of the output `.db` and its input be in the same directory. But that's not the case here. I think `mondo-ingest.db` should go into `tmp/` with the other `.db` files, but its source `mondo-ingest.owl` is not in `tmp/` for some reason.

How it works: So this goal basically just moves `mondo-ingest.db` to `tmp/mondo-ingest.db` after `mondo-ingest.db` is created through the `%.db` wildcard goal.